src/LiveDevelopment/Documents/HTMLDocument.js

```diff
@@ -326,8 +327,9 @@ define(function HTMLDocumentModule(require, exports, module) {
         // edit this file or set a conditional breakpoint at the top of this function:
         //     "this._debug = true, false"
         if (this._debug) {
```

Comment:
Why don't we just put `this._debug = false;` near the top of this file and leave the rest of this in place.